classes/view.php

```diff
@@ -257,13 +257,10 @@ protected function process_file($file_override = false)
 				// Load the view within the current scope
 				include $__file_name;
 			}
-			catch (\Exception $e)
+			finally
```

Comment:
Why did you do this? This breaks everything, on all PHP versions !

Reply:
I think finally is the better way than catching all exceptions ... ah, it will pass when no exception was thrown. It's my fault.
I requested revert PR #2160. Please merge it.